src/app/components/mnoe-impersonate-button/mnoe-impersonate-button.coffee

```diff
@@ -31,6 +31,17 @@
         url = url + "&dhbRefId=#{vm.organizationId}"
       $window.location.href = url
 
+    vm.openRequestModal = () ->
+      modalOptions =
+        closeButtonText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.cancel'
+        actionButtonText: 'mnoe_admin_panel.dashboard.users.widget.local_list.request_access'
+        headerText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.request_access_title'
+        bodyText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.body'
+
+      MnoConfirm.showModal(modalOptions).then( ->
+        vm.requestAccess()
```

Comment:
`vm.requestAccess()` should be passed as an `actionCb` in the `modalOptions`

Reply:
mnoeImpersonateModal was the modal replaced by the use of MnoConfirm, just forgot to delete the files.